app/src/main/java/com/bakkenbaeck/sol/extension/LifecycleExtensions.kt

```diff
@@ -0,0 +1,12 @@
+package com.bakkenbaeck.sol.extension
+
+import android.arch.lifecycle.LifecycleOwner
+import android.arch.lifecycle.LiveData
+import android.arch.lifecycle.Observer
+
+inline fun <reified T> LifecycleOwner.observe(field: LiveData<T>,
+                                              crossinline callback: (T) -> Unit) {
+    field.observe(this, Observer {
+        if (it != null) callback(it)
+    })
+}
```

Comment:
I'd personally remove this, no idea why this is needed as an extension function. Seems to just be syntactical sugar.